frontend/pulse3d/src/components/interactiveAnalysis/InteractiveAnalysisModal.js

```diff
@@ -725,12 +731,19 @@ export default function InteractiveWaveformModal({ selectedJob, setOpenInteracti
         </>
       )}
       <ModalWidget
-        open={["status", "dataFound", "pulse3dWarning", "duplicate"].includes(modalOpen)}
+        open={["status", "dataFound", "pulse3dWarning"].includes(modalOpen)}
         buttons={modalLabels.buttons}
         closeModal={handleModalClose}
         header={modalLabels.header}
         labels={modalLabels.messages}
       />
+      <ModalWidget
+        open={duplicateModalOpen}
+        buttons={constantModalLabels.duplicate.buttons}
+        closeModal={(userWarningChoice) => handleDuplicatesModalClose(userWarningChoice === 1)}
```

Comment:
You don't need to do this, `1` is a truthy value and `0` is falsey so they work fine as conditions in `if` statements